front/app/components/UI/QuillEditor/index.tsx

```diff
@@ -561,6 +571,22 @@ const QuillEditor = memo<Props & InjectedIntlProps>(
       // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [focussed, onFocus, onBlur]);
 
+    useEffect(() => {
```

Comment:
I had to add the placeholder for the alt image in this way to make sure we can use intl for it.